Tasks/PublishCodeCoverageResults/task.json

```diff
@@ -28,7 +28,8 @@
             "defaultValue": "JaCoCo",
             "helpMarkDown": "Select the tool with which code coverage summary files have been generated",
             "options": {
-                "JaCoCo": "JaCoCo"
+                "JaCoCo": "JaCoCo",
```

Comment:
Today all the help texts are with respect to jacoco. See if there is a way to give different help for different tools.